guides/core/user-interfaces.md

```diff
@@ -61,7 +61,7 @@ as a deployment wrapper around the `my_app_ui` project.
       # Dependencies for all targets
       {:nerves, "~> 1.7.0", runtime: false},
       # ...
-      {:my_app_ui, path: "../my_app_ui", targets: @all_targets, env: Mix.env()},
+      {:my_app_ui, path: "../my_app_ui", targets: [:host | @all_targets], env: Mix.env()},
```

Comment:
```suggestion
      {:my_app_ui, path: "../my_app_ui", env: Mix.env()},
```
It seems like the `:targets` field could be dropped if `:my_app_ui` is going to be pulled in for everything. @mnishiguchi or @jjcarstens - do either of you remember why `:targets` was added in the first place?

Reply:
I just tested this and it works fine with `:targets` field dropped and it worked fine. Unless there is a reason to specify it explicitly, I agree it should be dropped.

Reply:
@fhunleth I verified that [our official phoenix example](https://github.com/nerves-project/nerves_examples/blob/8cc860beefe2e389e1dccd1efb4ef8f43b1a794d/hello_phoenix/firmware/mix.exs#L56) does not have the `:target` field there.

```elixir
      {:ui, path: "../ui", env: Mix.env()},
```